core/src/miner/mem_pool_types.rs

```diff
@@ -433,15 +433,13 @@ impl MemPoolMinFees {
             Action::SetShardUsers {
                 ..
             } => self.min_set_shard_users_transaction_cost,
-            Action::Custom {
-                ..
-            } => self.min_custom_transaction_cost,
             Action::ShardStore {
                 ..
             } => {
                 // FIXME
                 0
             }
+            _ => self.min_custom_transaction_cost,
```

Comment:
How about creating match cases for each Actions Instead of using `_` pattern?
If we use `_` pattern in our code, it is hard to find the match expression when we add a new enum variant in the Action.

However, if this code will be removed and all transactions will be moved to the Module level, it's good to use `_` pattern here for temporary.

Reply:
https://github.com/CodeChain-io/foundry/pull/292 will remove the minimum cost. I'll rebase it on #292.